documentation/markup.rst

```diff
@@ -1019,13 +1019,19 @@ units as well as normal text:
    feature, or a part of it, to the library or C API.  When this applies to an
    entire module, it should be placed at the top of the module section before
    any prose.
+   When a new attribute, function, or method is added, make sure a
+   ``versionadded`` directive is added.
```

Comment:
```suggestion
   When a new attribute, function, or method is added, a
   ``versionadded`` directive should be present at the end of its
   documentation block.
```
Also describe explicitly where to put the directive.

Reply:
That's an improvement! Thanks.